src/main/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM frolvlad/alpine-oraclejdk8:slim
+VOLUME /tmp
+ADD test-env-runner-0.1.0.jar app.jar
+RUN sh -c 'touch /app.jar'
```

Comment:
You can use a RUN command to "touch" the jar file so that it has a file modification time (Docker creates all container files in an "unmodified" state by default). Any static content (e.g. "index.html") would require the file to have a modification time.